src/parser/expression.js

```diff
@@ -714,8 +716,17 @@ pp.parseObj = function (isPattern, refShorthandDefaultPos) {
       prop.type = isPattern ? "RestProperty" : "SpreadProperty";
       node.properties.push(prop);
       if (isPattern) {
-        this.expect(tt.braceR);
-        break;
+        const position = this.state.start;
+        if (firstRestLocation !== null) {
+          this.unexpected(firstRestLocation, "Cannot have multiple rest elements in destruction expression");
```

Comment:
Maybe `"Cannot have multiple rest elements in a destructuring assignment"` or `"Cannot have multiple rest elements when destructuring"`?